imessage/ios/ipc.go

```diff
@@ -407,7 +407,13 @@ func (ios *iOSConnector) SendFile(chatID, filename string, pathOnDisk string, re
 }
 
 func (ios *iOSConnector) SendFileCleanup(sendFileDir string) error {
-	return os.RemoveAll(sendFileDir)
+	go func() {
+		// Random sleep to make sure the message has time to get sent
+		time.Sleep(60 * time.Second)
+		ios.log.Warnln("Deleted temporary file from file transfer, hoping file upload succeeded by now.")
```

Comment:
It would definitely have helped me troubleshoot this and I feel like the "random 60 seconds" are also a pointer that I won't be the last person to debug this.